site/app/libraries/database/DatabaseQueries.php

```diff
@@ -2248,10 +2276,22 @@ public function getTotalComponentCount($g_id) {
         return intval($this->course_db->row()['cnt']);
     }
 
+    /**
+     * Returns the total number of verified components found in the database
+     *
+     * @param  string $g_id
+     * @return int
+     */
+    public function getTotalVerifiedComponentCount(string $g_id): int {
```

Comment:
There's no need to specify PHPDoc type annotations when you already have native type annotations.
```suggestion
     */
    public function getTotalVerifiedComponentCount(string $g_id): int {
```